### PR TITLE
Gh 1775/feature/promo modal metrics pings

### PR DESCRIPTION
**Pull request by wlycdgr**
Implements Plus and Insights promo modal metrics pings. Details in GH-1775.

* [X] Have you followed the guidelines in [CONTRIBUTING.md](../CONTRIBUTING.md)?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [X] Have you added an explanation of what your changes do?
* [X] Does your submission pass tests?
* [X] Did you lint your code prior to submission?

### CODE REVIEW

File: app/panel/components/Panel.jsx [hunk 1]
```diff
@@ -275,6 +295,8 @@ class Panel extends React.Component {
 
 		sendMessage('promoModals.sawInsightsPromo', '', 'metrics');
```
Comment (christophertino) [inline app/panel/components/Panel.jsx]: what does the 'metrics' argument do here?  looks like the other message calls use the format `sendMessage('promoModals.sawPlusPromo', {});`

File: app/panel/components/Panel.jsx [hunk 2]
```diff
@@ -211,24 +217,36 @@ class Panel extends React.Component {
...
-		// TODO send metrics ping
+	_handlePromoSelectBasicClick = (location) => {
 		this.props.actions.togglePromoModal();
+
+		sendMessage('ping', 'promo_modals_select_basic_panel');
 	};
 
-	_handlePromoSelectPlusClick = () => {
-		// TODO send metrics ping
+	_handlePromoSelectPlusClick = (location) => {
```
Comment (christophertino) [inline app/panel/components/Panel.jsx]: Does ESLint throw an unused arg error here?

File: app/panel/components/Panel.jsx (hunk 1 above)
Comment (wlycdgr) [reply app/panel/components/Panel.jsx]: Forgot to ask Ben to change this when we did the Insights promo modal PR. Removing.

File: app/panel/components/Panel.jsx (hunk 2 above)
Comment (wlycdgr) [reply app/panel/components/Panel.jsx]: Thought I needed this, realized I didn't, but forgot to remove. Removing. Also removed unused modal arg from _handlePromoSignInClick

**Pull request merged**